bedgraph: Add BED to bedGraph converter

JureZmrzlikar opened the pull request.

tomazc commented:
@JureZmrzlikar: please, reformat the commit messages according to the [contributing guidelines](http://icount.readthedocs.io/en/latest/contributing.html#commit-and-pr-messages).

JureZmrzlikar commented:
@tomazc , i fixed the commit messages. Currently, i added just `name` and `description` parameters into bedGraph header. Do you think it makes sense to also add any other ones? Which ones did you use in the "old" iCount?

tomazc commented on iCount/files/bedgraph.py:
```diff
@@ -0,0 +1,59 @@
+
+    LOGGER.info('Converting %s to %s', bed, bedgraph)
+    with open(bedgraph, 'wt') as outfile:
+        print(' '.join(map(str, header)), file=outfile)
```
@JureZmrzlikar: why use `print` instead of `.write` to write to files?

JureZmrzlikar replied:
Why not? Isn't this is a matter of preference?
https://stackoverflow.com/questions/8598228/f-write-vs-print-f

tomazc replied:
For consistency sake. Everywhere else we use `.write`. Can you please use it here as well.

JureZmrzlikar replied:
Of course, no problem.
